packages/react-core/src/components/NotificationBadge/examples/NotificationBadge.md

```diff
@@ -29,3 +29,10 @@ You can display a number within the notification badge by passing in the `count`
 
 ```ts file='./NotificationBadgeWithCount.tsx'
 ```
+
+### With animation
+
+You can animate the icon within the notification badge by passing in the `hasNotify` property. In this example, the notification badge icon will only animate if the notification count is greater than 0.
```

Comment:
```suggestion
You can animate the icon within the notification badge by passing in the `shouldNotify` property. In this example, the notification badge icon will only animate if the notification count is greater than 0.
```

Reply:
thanks @thatblindgeye and @edonehoo, updated the md file accordingly